### DISCUSSION TITLE
shadcn-ui init not configuring properly

### DISCUSSION THREAD

**Discussion by Dario-Soatto** [shadcn-ui/ui#3191] [General]
When I run the command npx shadcn-ui@latest init I expect a series of queries; however, I only get a subset of the queries, and as a result, I don't have a properly configured tailwind.

<img width="583" alt="Screenshot 2024-03-26 at 1 48 38 AM" src="https://github.com/shadcn-ui/ui/assets/96464213/27941c62-98c0-462d-ac19-31576ea12221">

**Comment by ezeparziale** [comment]
Now it detects it automatically: globals.css and tailwind.config.ts/js
